catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/RecyclerViewFragment.java

```diff
@@ -322,13 +333,18 @@ private void startActionMode(@ActionModeType int type) {
 
 	@Override
 	public void onSelectionChanged(int selectedItemCnt) {
+		int numberMergeSelections = adapter.numberMergeSelections;
+		if (adapter.isMergeSelection && selectedItemCnt >= numberMergeSelections) {
+			ToastUtil.showError(getContext(), "We can only merge " + numberMergeSelections + " projects");
```

Comment:
is there a reason that this is displayed everytime? since this is shown when the second project is selected, it will always be there.